prefect_airbyte/client.py

```diff
@@ -172,25 +174,20 @@ async def trigger_manual_sync_connection(
             response = await client.post(
                 get_connection_url, json={"connectionId": connection_id}
             )
-            if response.status_code == 200:
-                job_id = response.json()["job"]["id"]
-                print(response.json())
-                job_created_at = response.json()["job"]["createdAt"]
-                return job_id, job_created_at
-            elif response.status_code == 404:
-                # connection_id not found
-                self.logger.warning(
-                    f"Connection {connection_id} not found, please double "
-                    f"check the connection_id ..."
-                )
+            response.raise_for_status()
+            job_id = response.json()["job"]["id"]
+            job_created_at = response.json()["job"]["createdAt"]
+            return job_id, job_created_at
```

Comment:
```suggestion
            response.raise_for_status()
      
            job = response.json()["job"]
            job_id = job["id"]
            job_created_at = job["createdAt"]
            return job_id, job_created_at
```

I think the `Returns:` docstring section needs reformatting